### ISSUE TITLE
moth data: location name can be used for different sites

### ISSUE THREAD

**Issue by chrisvanswaay** [BiologicalRecordsCentre/ABLE#479]
@JimBacon @kazlauskis 
I notice in the moth data that the same location-name can be used for different locations: the name 'Moje' is linked to 5 different locations. Now this is bound to produce errors in entering the data or in the analysis. Could it be made impossible to use the same name for different locations?

**Comment by chrisvanswaay** [comment]
As a consequence I cannot distinguish between traps in the downloads. The only way I can link between the samples and the occurrences is via Location (which can be the same for different traps)
Actually the main problem is that there is not a locations_id in the moth-downloads. That would already solve most of the problems.
Ideally there would be four downloads:

Trap
- trap_id (unique)
- type of trap
- type of lamp

location
- location_id (or whatever id you use), needed to link to a sample_id (unique)
- Location_name the recorder uses (if there is a location_id, the same name can be used for different traps, it is just a string to be able to communicate with the recorder)
- trap_id
- lat/long

sample
- sample_id (unique)
- location_id
- Recorder name
- date
- comment

occurrence
- sample_id
- occurrence_id (unique)
- species name
- count

**Comment by DavidRoy** [comment]
@johnvanbreda can you add location_id to the download

**Comment by johnvanbreda** [comment]
@chrisvanswaay I've now added location ID to both the moth trap downloads.

**Comment by johnvanbreda** [comment]
@DavidRoy do we need to consider the other suggestions for downloads above?

**Comment by chrisvanswaay** [comment]
Thanks!

**Comment by DavidRoy** [comment]
@JimBacon can you pick up the other downloads required?

**Comment by JimBacon** [comment]
Yes but it will be some weeks before it gets to the top of my list.
